packages/cloud/src/routes/index.ts

```diff
@@ -1,13 +1,15 @@
 import { createRouter } from '@withtyped/server';
 
+import { ServicesLibrary } from '#src/libraries/services.js';
 import { TenantsLibrary } from '#src/libraries/tenants.js';
 import type { WithAuthContext } from '#src/middleware/with-auth.js';
 import { Queries } from '#src/queries/index.js';
 
+import { servicesRoutes } from './services.js';
 import { createTenants } from './tenants.js';
 
-const router = createRouter<WithAuthContext, '/api'>('/api').pack(
-  createTenants(new TenantsLibrary(Queries.default))
-);
+const router = createRouter<WithAuthContext, '/api'>('/api')
+  .pack(createTenants(new TenantsLibrary(Queries.default)))
+  .pack(servicesRoutes(new ServicesLibrary(Queries.default)));
```

Comment:
it'll be great if we can align naming. i'm ok with `tenantsRoutes`

Reply:
updated